akka-actor/src/main/scala/akka/util/ConstantFun.scala

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright (C) 2015-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.util
+
+import akka.annotation.InternalApi
+import akka.japi.function.{ Function ⇒ JFun, Function2 ⇒ JFun2 }
+import akka.japi.{ Pair ⇒ JPair }
+
+/**
+ * INTERNAL API
+ */
+@InternalApi private[akka] object ConstantFun {
```

Comment:
That's not the right conclusion, people may want to just upgrade an Akka version in their Akka HTTP 10.0.6, for example if Akka has a critical bugfix they should be able to just upgrade that. Thus, I don't think we can just move it - it has to be a careful move away (copy the class, deprecate it etc).